### PR TITLE
Add Open Source Section for React Version #1151 solved

### PR DESCRIPTION
**Pull request by tusharjain365**
- page is available at /experience
image

- Data is dynamically fetched into the file via openSource file
- all the images are in assets/images/opensource/
image

- Ui is kept exactly same except hackathon section which is made using react-carousel library.
image

@smaranjitghose kindly merge pull request.

### CODE REVIEW

File: src/pages/Experience/Experience.js [hunk 1]
```diff
@@ -1,34 +1,233 @@
...
-            <h1>Hello, Experience</h1>
-        </div>
-    );
-}
-
-export default Experience;
+import React from 'react';
+import { Helmet } from "react-helmet";
+import styles from './Experience.module.css';
+import flipkart from '../../assets/images/opensource/flipkart.jpg';
```
Comment (anushbhatia) [inline src/pages/Experience/Experience.js:4]: maintain another js file similar to opensource.js for work experience too

File: src/pages/Experience/Experience.module.css [hunk 2]
```diff
@@ -0,0 +1,877 @@
...
+  }
+}
+
+@media (min-width: 800px) {
+  .page-content {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+/* img {
```
Comment (anushbhatia) [inline src/pages/Experience/Experience.module.css:482]: remove this too

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
Also adjust the images as they are not in right resolution and feel compressed in a bot

File: src/pages/Experience/Experience.js (hunk 1 above)
Comment (tusharjain365) [reply src/pages/Experience/Experience.js:4]: Ok, will do

File: src/pages/Experience/Experience.module.css (hunk 2 above)
Comment (tusharjain365) [reply src/pages/Experience/Experience.module.css:482]: Will also do that.

**Comment by tusharjain365** [comment]
@anushbhatia Kindly now take a look at it.
All the changes requested are done

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
Add some padding here

**Comment by tusharjain365** [comment]
@anushbhatia kindly have a look and kindly let me know if this is satisfied.

**Comment by anushbhatia** [comment]
Awesome work @tusharjain365. Keep it Up

**Pull request merged**